src/frame/endpoints/PolicyApiEndpoints.ts

```diff
@@ -0,0 +1,56 @@
+import {ErrorCodes} from "../../Constants";
+import * as ApiRequest from "../ApiRequest";
+import ApiState from "../ApiState";
+import Future from "futurejs";
+import {SDKError} from "../../shim";
+import {UserOrGroup, Policy} from "ironweb";
+
+export type UserOrGroupWithKey = UserOrGroup & {masterPublicKey: PublicKey<string>};
+
+interface PolicyApplyResponse {
+    usersAndGroups: UserOrGroupWithKey[];
+    invalidUsersAndGroups: UserOrGroup[];
+}
+
+/**
+ * Generate signature message from current user state
+ */
+function getSignatureHeader() {
+    const {segmentId, id} = ApiState.user();
+    return ApiRequest.getRequestSignature(segmentId, id, ApiState.signingKeys());
+}
+
+/**
+ * Create the policy apply request based on the `policy`.
+ * @param policy The policy to be applied.
+ */
+export function applyPolicy(policy: Policy) {
```

Comment:
I don't think this function needs to be exported, it shouldn't be used outside this file.

Reply:
It's used in the tests.